modules/post/linux/gather/checkvm.rb

```diff
@@ -145,6 +168,48 @@ def run
       end
     end
 
+    # Check BIOS Name
+    if !vm
+      bios_vendor = read_file('/sys/devices/virtual/dmi/id/bios_vendor')
```

Comment:
For my setup this doesn't match my string; but the internet says this is what gets shown on EC2 instances https://github.com/ansible/ansible/issues/35051

For my dom0 'host' machine running Xen's hypervisor, but with virtualbox, I get `innotek GmbH`

And for the domU 'guest' machine, I don't have this file

Reply:
Worse case, this is a best-effort module that can be improved even more in a followup PR :)